### ISSUE TITLE
Running SDAPS through a website

### ISSUE THREAD

**Issue by maddy-jp** [sdaps/sdaps_web#23]
Can we have SDAPS Web running on the server where users from different locations could scan the questionnaires and send the CSV with windows and/or Linux PC by logging in using the website?

**Comment by TheRadialActive** [comment]
Yeah, that's the main idea of sdaps_web, but I am afraid to tell you, that sdaps_web is far from usable.

**Comment by maddy-jp** [comment]
thanks for the reply @TheRadialActive .........oh to make the benefits of SDAPS reach to a larger audience is it possible to have it as docker-container running on the server that might help achieve our goal? can it work for windows users then.......we do not have even clear stepwise instructions for installation and basic stand-alone working for ubuntu......i will try and make it so that at least we have something to start with
